mmdet3d/datasets/transforms/transforms_3d.py

```diff
@@ -190,6 +190,8 @@ def _flip_on_direction(self, results: dict) -> None:
         """
         if 'flip' not in results:
             cur_dir = self._choose_direction()
+        elif not results['flip']:
```

Comment:
This is a risky change to make sure it does not cause bc-breaking of tasks such as mono det, multi-modality det.

Reply:
I have checked and added comment for this line of code.